### PR TITLE
Tasks 442-452 : opus_paracrawl and com_qa datasets

### PR DESCRIPTION
**Pull request by Arutselvan**

### CODE REVIEW

**Comment by swarooprm** [comment]
Find the reviews of these tasks here: https://github.com/swarooprm/CSE576HW/pull/8

**Comment by danyaljj** [comment]
@kuntalkumarpal wondering if you can review this PR?

File: tasks/task444_com_qa_question_paraphrases_answer_generation.json [hunk 1]
```diff
@@ -0,0 +1,9432 @@
...
+    "Contributors": [
+        "Arut Selvan Dhanasekaran"
+    ],
+    "Source": [
+        "https://huggingface.co/datasets/com_qa"
+    ],
+    "Categories": [
+        "Answer Generation"
+    ],
+    "Definition": "Given a question and it's paraphrases, answer the question. The answer should exactly answer all the questions given without any ambiguity. Don't give partial answers.",
```
Comment (kuntalkumarpal) [inline tasks/task444_com_qa_question_paraphrases_answer_generation.json]: Typo : it's -> its

**Comment by kuntalkumarpal** [comment]
For task445_com_qa_wikipedia_answer_generation.json:

"Definition": "Given a question and **it's** paraphrases, give a Wikipedia page link that might answer all the questions. **The Wikipedia page should be directly related to the question**"

The highlighted part of the question is not clear based on the below examples. What do you mean by this ? Do you mean some phrases should overlap between question and answer ? or The page should have the answer ?

Positive Example : 
`{
            "input": "Questions: ['what is the fourth biggest city in scotland?']",
            "output": "https://en.wikipedia.org/wiki/dundee",
            "explanation": "The linked wikipedia page is directly related to the given question"
        }`

Negative Example: 
`{
            "input": "Questions: ['What is the name of the Internationa airport in Phoenix?', 'Name any one of Arizona's international airports']",
            "output": "https://en.wikipedia.org/wiki/Arizona_State_University",
            "explanation": "The wikipedia page is not directly related to the given questions even though the answer might be present in the page"
        }`

**Comment by kuntalkumarpal** [comment]
In Task 448 : Eng to Tagalog Positive Example

`{
            "input": "English sentence: A manufacturer of led flashing light 5 watt at GuangDong China.",
            "output": "Ang tagagawa ng LED panggayak liwanag sa Guangdong China.",
            "explanation": "The output is the exact Tagalog translation of the input English text."
        }`

I think it misses 5 watt in the translation. Can you once verify if it is marked as correct/positive translation in the dataset by the author? 

Same example for task 451

**Comment by kuntalkumarpal** [comment]
Rest of the Task looks good to me

**Comment by swarooprm** [comment]
@Arutselvan, wondering if you can address the reviewer comments?

**Comment by Arutselvan** [comment]
> @Arutselvan, wondering if you can address the reviewer comments?

Yeah working on the changes, will update the PR ASAP.

**Comment by swarooprm** [comment]
> > @Arutselvan, wondering if you can address the reviewer comments?
> 
> Yeah working on the changes, will update the PR ASAP.

Wondering if you have any update on this? @Arutselvan

**Comment by Arutselvan** [comment]
> > > @Arutselvan, wondering if you can address the reviewer comments?
> > 
> > 
> > Yeah working on the changes, will update the PR ASAP.
> 
> Wondering if you have any update on this? @Arutselvan

I've made the changes based on the comments provided. Please review @swarooprm

**Comment by swarooprm** [comment]
@Arutselvan, thanks! Based on reviewer comment above, everything looks good except task445:
Even though you have access to question paraphrases, keep only one question in the input of instances as well as examples

Definition. The second line seems redundant.
I would keep this as definition 
"Given a question, generate a wikipedia page link that contains the answer to the question""

In the negative examples, use irrelevant output: instead of Arizona state university, add link for something else may be Canada, similarly instead of worldwar-2 which is similar to worldwar-1, provide link for say an scientist, similarly for 3rd example.

Let me know once you do these changes so that your tasks can get accepted.

**Comment by Arutselvan** [comment]
@swarooprm I've modified task 445 with single question as input Please review.

**Review by swarooprm — APPROVED** [review]
Thanks @Arutselvan! Congratulations on the acceptance of your tasks to our repo.

**Pull request merged**